lib/src/main/java/vmath/Vec3.java

```diff
@@ -16,56 +33,125 @@ public Vec3(float x, float y, float z) {
         this.z = z;
     }
 
+    /**
+     * Returns the x component of this vector.
+     *
+     * @return The x component.
+     */
     public float x() {
         return x;
     }
 
+    /**
+     * Returns the y component of this vector.
+     *
+     * @return The y component.
+     */
     public float y() {
         return y;
     }
 
+    /**
+     * Returns the z component of this vector.
+     *
+     * @return The z component.
+     */
     public float z() {
         return z;
     }
 
+    /**
+     * Calculates and returns the magnitude of this vector.
+     *
+     * @return The magnitude of the vector.
+     */
     public float mag() {
-        return (float) Math.sqrt(x*x + y*y + z*z);
+        return (float) Math.sqrt(x * x + y * y + z * z);
```

Comment:
This PR brings unwanted changes to the code. By the [conventions](https://github.com/toksaitov/vmath/wiki/Code-Style), we are required to not put spaces around multiplication/division operators.

Reply:
So what am I expected to do now? Am I supposed to re-do all the steps?

Reply:
> So what am I expected to do now? Am I supposed to re-do all the steps?

In all the lines where you accidentally changed the actual code, change the style to how things used to be. There are few lines where changes are needed, you may do this manually.